vllm/distributed/parallel_state.py

```diff
@@ -39,9 +39,9 @@
 
 
 def init_distributed_environment(
-    world_size: int,
-    rank: int,
-    distributed_init_method: Optional[str] = None,
+    world_size: int = -1,
+    rank: int = -1,
+    distributed_init_method: str = "env://",
```

Comment:
@youkaichao shouldn't the type here remain as `Optional[str]` since presumably it could still be valid to pass `None`?

Reply:
for compatibility, yes, `Optional[str]` is better. but does anyone really explictly pass `None`? I don't want to do an extra check `if distributed_init_method  is None` 🤣